### ISSUE TITLE
Doesn't work on docker.

### ISSUE THREAD

**Issue by Bitwolfies** [jellyfin/jellyfin-plugin-anime#89]
Running stable, 1,6.2, wont auto identify, and manual identification returns zero results.

**Comment by StillLoading** [comment]
Are you using AniDB ? I've had the same issue before the fix in #87, but it looks like there has been no new release of the plugin after that.

**Comment by Bitwolfies** [comment]
Yes, anidb, latest version of the plugin, 9.0.0.0.0 or something along those lines.

**Comment by StillLoading** [comment]
Okay, then you'll have to wait till the 10.0 release or compile the current master yourself to get the fix.
